woocommerce/wc-admin
# Add route and layout for unmatched path

Adds a route for unmatched paths and routes where the user does not have the capability.

**Question:** This behavior mimics that of what exists in WP core, by showing the `Sorry, you are not allowed to access this page.` message regardless of whether or not permission is an issue or if the page simply does not exist.  Do we want to continue replicating this behavior or have separate messages for genuinely not found versus missing capabilities?

### Screenshots

<img width="1165" alt="Screen Shot 2021-08-12 at 10 51 30 AM" src="https://user-images.githubusercontent.com/10561050/129219865-87b109fa-a1ec-43e7-9488-305ef6e59dcf.png">


### Detailed test instructions:

1. Create a user that can `read`, `view_woocommerce_reports`, but cannot `manage_woocommerce`.
2. Visit `wp-admin/admin.php?page=wc-admin`
3. Note that the error message is shown
4. Visit a bad analytics report path.  E.g., `wp-admin/admin.php?page=wc-admin&path=/analytics/nope`
5. Check that the error message is shown
